js/src/common/Application.js

```diff
@@ -199,12 +199,18 @@ export default class Application {
 
     // Add a class to the body which indicates that the page has been scrolled
     // down.
-    new ScrollListener((top) => {
+    const scrollListener = new ScrollListener((top) => {
       const $app = $('#app');
       const offset = $app.offset().top;
 
       $app.toggleClass('affix', top >= offset).toggleClass('scrolled', top > offset);
```

Comment:
Let's add a comment explaining that the affix class is used for when custom headers are displayed above Flarum's header.

Reply:
Well, affix itself is used so the header is sticky when scrolled down. But yeah I'll expand that comment a bit, thanks for the suggestion!